### PR TITLE
 Add milestone filtering to PR dashboard.

### PR DESCRIPTION
**Pull request by rmmh**
This should be helpful during releases.

Test deployment running here: https://test-dot-k8s-gubernator.appspot.com/pr

### CODE REVIEW

File: gubernator/Makefile [hunk 1]
```diff
@@ -20,7 +20,7 @@ PROJECT ?= k8s-gubernator
 .PHONY: migrate versions deploy cron all
 all: versions
 
-deploy: cron
+deploy:
```
Comment (BenTheElder) [inline gubernator/Makefile]: this change seems unrelated?

**Comment by michelle192837** [comment]
/lgtm

For redundancy :P
